lib/mj-librsvg.js

```diff
@@ -0,0 +1,26 @@
+var rsvg = require('librsvg').Rsvg;
+var Readable = require('stream').Readable;
+
+exports.svg2png = function(svg, result, scale, width, height, callback, check) {
+  var s = new Readable();
+  var svgRenderer = new rsvg();
```

Comment:
Note that this still allows you to keep the block on L11 (error checking), because the `rsvg` object is a stream itself.